### ISSUE TITLE
Add a new organization to ROR: Research Centre for Open Scholarly Metadata

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#63]
Summary of request: Add a new organization to ROR

Name of organization to be added | Research Centre for Open Scholarly Metadata

Organization website | https://openscholarlymetadata.org/

Briefly explain why you are requesting a ROR ID for this organization and how the ROR ID will be used. | Scholarly infrastructures such as CDL, Crossref and DataCite all have RORs. Inspired by PIDapalooza 2021, following these examples, and having just requested a ROR for OpenCitations, I request a ROR for OpenCitations' host institution The Research Centre for Open Scholarly Metadata.

Provide DOI(s) or links for research associated with this organization, such as published articles that include the organization's name as an affiliation. If DOIs or links are not available, describe the research-related activities of this organization. | https://doi.org/10.1162/qss_a_00023

Other relevant metadata about the organization (e.g., name variants in other languages, related records in ROR, links to external records such as Wikidata) | https://github.com/openscholarlymetadata, https://workshop-oc.github.io/

**Comment by carlyrobinson** [comment]
May be associated with two separate organizations - OpenCitations (which we've already approved to add to ROR) and  University of Bologna within the Department of Classical Philology and Italian Studies. Used as an affiliation, but under this department at University of Bologna. Not sure if it should have separate ROR.

**Comment by mariagould** [comment]
Some additional comments from the original request that may shed light on the need for a new ROR ID and the center's relationship to OpenCitations:

> The reason that we need two RORs, one for OpenCitations and one for the Research Centre for Open Scholarly Metadata is as follows:
> OpenCitations is distinct from and pre-existed the Research Centre, having grown to become a valuable independent infrastructure from a small beginning as a research project in my lab.
> The Research Centre was created specifically for two purposes: (a) to provide an institutional affiliation, a legal financial centre, and a formal administrative 'home' for OpenCitations, and (b) to permit academic activities affiliated with OpenCitations, such as being the academic home for graduate students of the University of Bologna working on projects involving open citations.  The Research Centre is formally a part of the University of Bologna in a way that OpenCitations is not.
> In the medium-term future, OpenCitations may 'spin out' to become a non-profit foundation /association/Stiftung under direct governance by the wider academic community and other stakeholders, that is administratively and financially independent from the University of Bologna.  In that case, OpenCitations will continue with its stated mission of providing open bibliographic metadata and citations, and the Research Centre will continue as an integral part of the University of Bologna in its academic roles with respect to OpenCitations, and will probably by then also have expanded its activities to include other types of metadata-oriented research work, but it will no longer be administratively and financially responsible for OpenCitations.
> For these reasons of distinctiveness, it would be sensible if OpenCitations and the Research Centre for Open Scholarly Metadata were issued separate RORs from the start, rather than trying to disentangle them at some point down the line.  That was the basis of my thinking when I wrote.

**Comment by mariagould** [comment]
We believe this should be reviewed at a later point to determine if there is a use case for a separate ROR ID for this center.

**Comment by adambuttrick** [comment]
Affiliation usage in 2022 continues to be a leading affiliation for the University of Bologna:

https://arxiv.org/ftp/arxiv/papers/2205/2205.06764.pdf

Since this is essentially a request to create a new ROR ID contingent on the some kind of yet-to-occur restructuring, I'd advise we decline this request. They can resubmit in the event any of what they described in the above actually occurs.

**Comment by adambuttrick** [comment]
Situation is unchanged. Declining as out of scope.
